src/smif/convert/__init__.py

```diff
@@ -112,3 +114,52 @@ def _convert_intervals(self, data, from_temporal, to_temporal):
         converted = np.apply_along_axis(self.intervals.convert, 1, data,
                                         from_temporal, to_temporal)
         return converted
+
+
+class UnitConvertor(object):
+    """Handles the conversion of units for a numpy array of values
+
+    Arguments
+    ---------
+    unit_register: :class:`smif.convert.unit.UnitRegister`
```

Comment:
There are no arguments here.  Can safely remove this.

Reply:
Done